src/FlatSelect/components/Dropdown/Dropdown.tsx

```diff
@@ -71,27 +72,34 @@ const Dropdown = React.forwardRef(
 
 		React.useEffect(
 			() => {
-				itemsListSearch = [];
+				handleFocusElement(refSearch.current);
```

Comment:
Этой функции бы дать более релевантное имя
```suggestion
				setFocusOnElement(refSearch.current);
```